frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/redux/state-update/vm-settings-tab-state-update.ts

```diff
@@ -162,6 +168,71 @@ const osUpdater = ({ id, prevState, dispatch, getState }: UpdateOptions) => {
   if (!isWindows && windowsTools) {
     dispatch(vmWizardInternalActions[InternalActionType.RemoveStorage](id, windowsTools.id));
   }
+
+  const iCommonTemplates = iGetLoadedCommonData(state, id, VMWizardProps.commonTemplates);
+  const iTemplate = iCommonTemplates && iGetOSTemplate(iCommonTemplates, os);
```

Comment:
this is selecting the wrong template and then consequently checking wrong pvc

Reply:
@suomiy  what am I missing here ?
how do I select the currect one ?

Reply:
with `iGetRelevantTemplate` ?

Reply:
I'm using `iGetRelevantTemplate` now, but it's the same implementation, if `iGetOSTemplate` return the wrong template `iGetRelevantTemplate` will also return wrong template ?

Reply:
using `iGetRelevantTemplate` now